extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/devmode/ConfigDescriptionsManager.java

```diff
@@ -28,6 +28,9 @@ public class ConfigDescriptionsManager extends DevConsolePostHandler implements
     private final Set<String> devServicesProperties;
     private ClassLoader currentCl;
 
+    private static final String quotedDot = "\".\"";
```

Comment:
Static final attributes must be uppercased

Reply:
Wow, I am battling to get this right. Apologies :)